apps/carbon/app/routes/api+/trigger.ts

```diff
@@ -1,8 +1,28 @@
-import { createRemixRoute } from "@trigger.dev/remix";
+//import { createRemixRoute } from "@trigger.dev/remix";
 import { triggerClient } from "~/lib/trigger.server";
+import serverRuntime from "@remix-run/server-runtime";
+import type { ActionFunction } from "@remix-run/node";
 
 // Remix will automatically strip files with side effects
 // So you need to *export* your Job definitions like this:
-export * from "~/jobs";
+//export * from "~/jobs.server";
```

Comment:
Amazing work with this PR and thank you!

One thing - having this line commented out prevents jobs from running/reaching Trigger and it seems like it may have been a typo. 

```suggestion
export * from "~/jobs.server";
```

Reply:
Thanks!  I'll address this in my next commit.